framework/lib/components/tooltip/tooltip.tsx

```diff
@@ -176,7 +187,7 @@ export const Tooltip: React.FC<OurTooltipProps> = ({
         <P
           variant="14"
           sx={ {
-            color: !variant || variant === 'ai' || variant === 'default' ? 'text.inverted' : undefined,
+            color: !variant || variant === 'ai',
```

Comment:
what is the type for color, is it supposed to be true | false?

Reply:
You're right. Amend is on the way.